tests/test_atomic_requests.py

```diff
@@ -135,16 +147,25 @@ def test_api_exception_rollback_transaction(self):
     "'atomic' requires transactions and savepoints."
 )
 @override_settings(ROOT_URLCONF='tests.test_atomic_requests')
-class NonAtomicDBTransactionAPIExceptionTests(TransactionTestCase):
+class NonAtomicDBTransactionAPIExceptionTests(TestCase):
```

Comment:
Just checking - is there a reason this shouldn't be a transaction test case?

Reply:
what else needed to get this merged in 3.11?

Reply:
@rpkilby `TransactionTestCase` is not necessary because the transaction is cleanly handled under the test client.

Reply:
Would it make sense to explicitly wrap the test within an `atomic` block? It might more clearly convey the purpose of the test. At least, it should probably be documented in a comment that `TestCase` provides the transaction.

Additionally, is there any behavior that needs to be tested when non-atomic is outside an atomic block?

Reply:
Regardless, just some minor details. The PR basically looks good to me.

Reply:
> At least, it should probably be documented in a comment that TestCase provides the transaction.

IMO this is relatively basic knowledge for using the Django test framework and doesn't need a comment.

> Additionally, is there any behavior that needs to be tested when non-atomic is outside an atomic block?

Don't follow exactly what you're asking?